src/common/interfaces/schedule.ts

```diff
@@ -14,6 +14,8 @@ export interface Parameters {
   show_aging_table: boolean;
   status: string;
   clients: string[];
+  entity: string;
```

Comment:
Is it possible to type possible entities?

Reply:
Sure, the typo is included now.